xstate-chatbot/nodejs/src/machine/service/gis-service.js

```diff
@@ -74,6 +74,17 @@ class GisService {
     formdata.append('SewerageConnection', propertyDetails.sewageConnection);
     formdata.append('PropertyTax', propertyDetails.propertyId);
     formdata.append('OwnersName', propertyDetails.ownerName);
+
+    if(propertyDetails.image){
+      let filestoreId = await this.getFileForFileStoreId(propertyDetails.image);
+      var content = {
+        documentType: "PHOTO",
+        filestoreId:filestoreId
```

Comment:
The upload mechanism would not be like this. Based on their(NIC) API definition, you would have to test first using Postman and then running simple JS functions from the terminal to check how to upload a file.

Reply:
gis service updated

Reply:
gis service updated again